src/main/osd/osd.c

```diff
@@ -545,6 +548,18 @@ void osdInit(displayPort_t *osdDisplayPortToUse, osdDisplayPortDevice_e displayP
     }
 }
 
+#ifdef USE_GPS_LAP_TIMER
+void printLapTime(char* buffer, uint32_t time) {
+    if (time != 0) {
+        const int timeSeconds = time / 1000;
```

Comment:
(you can gain slightly better precision by rounding  - `time = (time + 5) / 10` )